lms/static/certificates/sass/_components.scss

```diff
@@ -562,3 +561,83 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: z-index(front);
+    background-color: palette(grayscale, trans); /*dim the background*/
```

Comment:
NIT: please use the sass syntax for comments (e.g. ``// dim the background``), so that this comment doesn't get passed into the production CSS.